src/openrct2/entity/Peep.cpp

```diff
@@ -1146,15 +1166,33 @@ void peep_problem_warnings_update()
         warning_throttle[6] = 4;
         if (gConfigNotifications.guest_warnings)
         {
-            News::AddItemToQueue(News::ItemType::Peeps, STR_PEEPS_GETTING_LOST_OR_STUCK, 27, {});
+            constexpr auto thoughtId = static_cast<uint32_t>(PeepThoughtType::CantFindExit);
+            News::AddItemToQueue(News::ItemType::Peeps, STR_PEEPS_GETTING_LOST_OR_STUCK, thoughtId, {});
         }
     }
     else if (lost_counter >= PEEP_LOST_WARNING_THRESHOLD)
     {
         warning_throttle[6] = 4;
         if (gConfigNotifications.guest_warnings)
         {
-            News::AddItemToQueue(News::ItemType::Peeps, STR_PEEPS_GETTING_LOST_OR_STUCK, 16, {});
+            constexpr auto thoughtId = static_cast<uint32_t>(PeepThoughtType::Lost);
+            News::AddItemToQueue(News::ItemType::Peeps, STR_PEEPS_GETTING_LOST_OR_STUCK, thoughtId, {});
+        }
+    }
+
+    if (warning_throttle[7])
+        --warning_throttle[7];
```

Comment:
Just to clarify on this hardcoded index, index 7 seems to be unused before this PR. The array `gPeepWarningThrottle` has a length of 16, but only the first 7 are being used. This array is not accessed anywhere else, other than for saving and loading.